ClusterManagement/Cluster_url_clarification.md

```diff
@@ -0,0 +1,23 @@
+### URL difference
+[Offcial doc](https://hadoop.apache.org/docs/r2.7.3/) lists 
+all the APIs for each hadoop relevant service. However, because of the Gateway policy, HDInsight sometimes uses different URL to get the info. 
+This doc is an URL clarifaction for cases we knew so far. We would keep this file updated once we see more cases.
+
+### MapReduce API
+* Apache Hadoop
+```
+ http://<proxy http address:port>/proxy/{appid}/ws/v1/mapreduce
```

Comment:
What port? This needs a little bit explaining to do. Will there work from outside the cluster? If so, what kind of tunneling should be setup.

Reply:
This is the way apache official is useing, i believe they are using port 19888. But we are not support this URL from HDI. The next one is the correct way to check for the info.